src/test/java/core/basesyntax/WriterServiceTest.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import org.junit.Test;
+
+public class WriterServiceTest extends FruitShopTest {
```

Comment:
The same (don't need to extend from other classes. And add some tests for other situations)

Reply:
And do this for all your test classes